tests/unit/test_network_connections.py

```diff
@@ -3670,6 +3707,93 @@ def test_path_to_glib_bytes(self):
         self.assertEqual(result.get_data(), b"file:///my/test/path\x00")
 
 
+class TestValidatorMatch(unittest.TestCase):
+    def setUp(self):
+        self.test_profile = {
+            "name": "test",
+            "type": "ethernet",
+            "ip": {
+                "dhcp4": False,
+                "address": "192.168.245.7/24",
+            },
+        }
+        self.validator = network_lsr.argument_validator.ArgValidator_DictConnection()
+
+    def test_match_path_empty_list(self):
+        """
+        I can remove a match path setting by setting it to empty list.
+        """
+        self.test_profile.update({"match": {"path": []}})
+        result = self.validator.validate(self.test_profile)
+        self.assertEqual({"path": []}, result.get("match", {}))
```

Comment:
please move the expected values to the second argument, also in the other tests

Reply:
Updated.